src/sparseml/pytorch/utils/exporter.py

```diff
@@ -419,7 +419,7 @@ def export_onnx(
     create_parent_dirs(file_path)
 
     module = deepcopy(module).cpu()
-    module.eval()
+    # module.eval()
```

Comment:
My conviction is that we should not have commented out code in the production code. If its commented out, it should probably get deleted.